core/src/main/java/com/linecorp/armeria/server/ServerErrorHandler.java

```diff
@@ -236,4 +236,8 @@ public AggregatedHttpResponse renderStatus(ServiceConfig config,
             }
         };
     }
+
+    default ServiceErrorHandler asServiceErrorHandler() {
```

Comment:
Could you add Javadoc?

Reply:
Could you also deprecate `ServerErrorHandler.orElse(...)` because it's not going to be used anymore?

Reply:
We need `orElse` method. Sorry about it. 😅